### DISCUSSION TITLE
Can't import skins into osu!lazer for Android

### DISCUSSION THREAD

**Discussion by 88572** [ppy/osu#26146] [Q&A] [answered]
While attempting to import a skin into osu!lazer on Android, osu! isn't listed in the apps that can open .osk files. How can I install skins?
![Screenshot_20231226-103425.jpg](https://github.com/ppy/osu/assets/82726593/f5d031be-1f21-4594-840b-911c3ded06c3)

**Comment by ItsShamed** [comment, marked as answer]
You **share** it, and osu! will be listed as an app you can share the skin to. 

https://github.com/ppy/osu/assets/39100084/1f06a826-72cc-4f9b-8cfa-4bbc5cecff2c

**Comment by 88572** [comment]
That makes sense, thanks 👍


________________________________
